src/assets/src/components/queue.tsx

```diff
@@ -549,6 +549,30 @@ export function QueuePage(props: PageProps) {
     const loginDialogVisible = errorSources.some(checkForbiddenError);
     const loadingDisplay = <LoadingDisplay loading={isChanging}/>
     const errorDisplay = <ErrorDisplay formErrors={errorSources}/>
+
+    let queueTitle: string;
+
+    if (!queue) {
+        // Queue data is not available or not yet loaded.
+        queueTitle = queue_id ? `Queue ${queue_id}` : "Loading Queue...";
+    } else {
+        // Queue data IS available.
+        const displayName = queue.name || (queue_id ? `Queue ${queue_id}` : "Unnamed Queue");
```

Comment:
Looks like in line 446, it checks for queue_id already and throws an error if it is not defined:
`if (queue_id === undefined) throw new Error("queue_id is undefined!");`

So maybe we can drop the `queue_id?` check?

Reply:
Yes, I agree, I will make those changes!